.changeset/eighty-taxis-wait.md

```diff
@@ -0,0 +1,16 @@
+---
+"astro": minor
+---
+
+Allow actions to be called on the server. This allows you to call actions as utility functions in your Astro frontmatter or server-side components.
```

Comment:
Does it also allow using them in endpoints?

Reply:
Yup! Just didn’t expect that use case as much. I’ll update the changeset